.gitignore

```diff
@@ -0,0 +1,4 @@
+.terraform
+.terraform.*
+terraform.*
+.git
```

Comment:
could you please, add newline at the end of file.

Reply:
Added!